modules/sdk-core/src/bitgo/keychain/keychains.ts

```diff
@@ -305,4 +310,121 @@ export class Keychains implements IKeychains {
       backupProvider: params.backupProvider,
     });
   }
+
+  /**
+   * It parses the JSON downloaded from the OVC for platform (BitGo),
+   * and creates a corresponding TSS BitGo key. It also returns the JSON that needs
+   * to be uploaded back to the OVCs containing the BitGo -> OVC shares.
+   * @param ovcOutputJson JSON format of the file downloaded from the OVC for platform
+   * @returns {BitGoKeyFromOvcShares}
+   */
+  async createTssBitGoKeyFromOvcShares(ovcOutputJson: OvcToBitGoJSON): Promise<BitGoKeyFromOvcShares> {
```

Comment:
A lot of the validation done here can be done via io-ts. 

We have io-ts available in sdk-core, lets instead create a strict codec and decode the json file using the codec.

Reply:
I need custom errors for each field that is missing, and io-ts decode will not support that and throw an arbitary error if it doesn't match the coded. In addition, i do already have a type defined for the JSON and if(xyz) will check for a non-empty string. I can add tests for some non-happy paths to test this.